lib/library.go

```diff
@@ -70,6 +70,24 @@ func ProcessContactCode(bundleString *C.char) *C.char {
 	return nil
 }
 
+// Get an X3DH bundle
+//export GetContactCode
+func GetContactCode(identityString *C.char) *C.char {
```

Comment:
From now on, it also needs to be added to `mobile/`.

Reply:
done!